app/assets/stylesheets/components/forms.scss

```diff
@@ -86,6 +86,7 @@
   &--disabled,
   &[disabled] {
     @extend %form-styling-disabled;
+    cursor: not-allowed;
```

Comment:
@ludwiczakpawel I want to propose changing the cursor if it's disabled so that the user knows they can't select it. What do you think about this update?

Reply:
Makes sense, go for it!